deploy/charts/origin-ca-issuer/README.md

```diff
@@ -0,0 +1,83 @@
+# origin-ca-issuser
+
+origin-ca-issuer is a Kubernetes addon to automate issuance and renewals of Cloudflare Origin CA certificats with cert-manager.
+
+## Prerequisites
+
+* Kubernetes 1.16+
+* cert-manager 1.0.0+
+
+## Installing the Chart
+
+Before installing the chart, you must first install [cert-manager](https://cert-manager.io/docs/installation/), and the origin-ca-issuer CustomResourceDefinition resources.
+
+```shell
+VERSION="v0.5.0"
+kubectl apply -f https://raw.githubusercontent.com/cloudflare/origin-ca-issuer/${VERSION}/deploy/crds/cert-manager.k8s.cloudflare.com_originissuers.yaml
+```
+
+To install the chart with the release name `my-release`:
+
+``` shell
+VERSION="v0.5.0"
```

Comment:
Does this do anything?

Reply:
Originally this was going to say `helm install` from GitHub, but that's not actually method.